Samples/IonicCapacitorAngularV2/ct-demo-ionic-angular-cap/src/app/home/home.page.ts

```
@@ -0,0 +1,266 @@
+        break;
+      case "Update(Add) Decrement Value":
+        CleverTap.profileDecrementValueBy("score", 30);
+        break;
+      case "Profile Location":
+        CleverTap.setLocation(19.232390, 72.827530);
+        break;
+      case "Get User Profile Property":
+        CleverTap.profileGetProperty("Name").then(r => this.presentToast("Property value of Name is " + r));
+        break;
+      case "onUserLogin":
+        let n, p;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)